xml/System.Windows/MessageBoxButton.xml

```diff
@@ -61,7 +61,9 @@ The value of the button that a user clicks is returned by <xref:System.Windows.M
       </ReturnValue>
       <MemberValue>2</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>
+          <para>The message box displays **Abort**, **Retry**, and **Ignore** buttons.</para>
```

Comment:
\<para> tags are unnecessary if there's only a single sentence/paragraph.

Reply:
Removed the extra tags